diff -Naur st-0.9.2/config.def.h myterm/config.def.h
--- st-0.9.2/config.def.h	2024-04-05 06:19:18.000000000 -0400
+++ myterm/config.def.h	2024-06-26 02:44:26.580346863 -0400
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "JetBrainsMonoNL:pixelsize=12:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
